### PR TITLE
Fix assertion in dns tests wrt changed docker json API

### PR DESCRIPTION
**Pull request by jrudolph**
Refs #27489

### CODE REVIEW

**Review by raboof — APPROVED** [review]
Okay

File: akka-actor-tests/src/test/scala/akka/io/dns/DockerBindDnsService.scala [hunk 1]
```diff
@@ -70,7 +70,7 @@ trait DockerBindDnsService extends Eventually { self: AkkaSpec =>
       })
 
     val creation = client.createContainer(containerConfig, containerName)
-    creation.warnings() should be(null)
+    creation.warnings() should be(null).or(be('empty))
```
Comment (raboof) [inline akka-actor-tests/src/test/scala/akka/io/dns/DockerBindDnsService.scala]: symbols are deprecated in Scala 2.13

Comment (jrudolph) [reply akka-actor-tests/src/test/scala/akka/io/dns/DockerBindDnsService.scala]: bummer

Comment (jrudolph) [reply akka-actor-tests/src/test/scala/akka/io/dns/DockerBindDnsService.scala]: fixed.

**Comment by jrudolph** [comment]
Currently failing all the nightlies / regular builds so would be good to merge soon.

**Review by johanandren — APPROVED** [review]

**Pull request merged**